### PR TITLE
ci: remove linux-arm64 support for ampd

### PR DESCRIPTION
**Pull request by cgorenflo**

### CODE REVIEW

File: .github/workflows/build-ampd-release.yaml [hunk 1]
```diff
@@ -81,21 +84,22 @@ jobs:
...
-            gpg --armor --detach-sign  "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
           else
-            brew install protobuf
             cargo build --release
             mkdir ampdbin
             mv "/Users/runner/work/axelar-amplifier/axelar-amplifier/target/release/ampd" "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
-            gpg --armor --detach-sign  "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
           fi
+          
+          gpg --armor --detach-sign  "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
```
Comment (milapsheth) [inline .github/workflows/build-ampd-release.yaml:102]: :+1:

Comment (milapsheth) [inline .github/workflows/build-ampd-release.yaml:102]:
```suggestion
          gpg --armor --detach-sign "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
```

File: .github/workflows/build-ampd-release.yaml [hunk 2]
```diff
@@ -104,22 +108,14 @@ jobs:
...
-            gpg --armor --detach-sign  "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
-          fi
+          
+          sudo apt-get install libssl-dev protobuf-compiler
+          
+          cargo build --release
+          mkdir ampdbin
+          mv "/home/runner/work/axelar-amplifier/axelar-amplifier/target/release/ampd" "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
+          
+          gpg --armor --detach-sign  "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
```
Comment (milapsheth) [inline .github/workflows/build-ampd-release.yaml:118]:
```suggestion
          gpg --armor --detach-sign "./ampdbin/ampd-$OS-$ARCH-$SEMVER"
```

File: .github/workflows/build-ampd-release.yaml [hunk 3]
```diff
@@ -188,7 +184,7 @@ jobs:
           cp -R ./ampdbin/. "./${version}/"
           echo "release-dir=./${version}" >> $GITHUB_OUTPUT
           echo "r2-destination-dir=./releases/ampd/" >> $GITHUB_OUTPUT
-          
+
```
Comment (milapsheth) [inline .github/workflows/build-ampd-release.yaml:188]:
```suggestion
```
extra newline

**Review by milapsheth — APPROVED** [review]

File: .github/workflows/build-ampd-release.yaml [hunk 4]
```diff
@@ -188,7 +184,7 @@ jobs:
...
```
Comment (cgorenflo) [inline .github/workflows/build-ampd-release.yaml:187]:
```suggestion
```

**Review by talalashraf — APPROVED** [review]
@apolikamixitos note here

**Pull request merged**